fragments/filter/README.md

```diff
@@ -932,3 +941,103 @@ filter=sentinel:data_coverage > 50 OR landsat:coverage_percent < 10 OR (sentinel
   }
 }
 ```
+
+### Example 10: Using BETWEEN
+
+The BETWEEN operator allows for checking if a numeric value is within a specified inclusive range.
+
+#### Example 10: cql2-text (GET)
+
+```http
+filter=eo:cloud_cover BETWEEN 0 AND 50
+```
+
+#### Example 10: cql2-json (POST)
+
+```json
+{
+  "filter": {
+    "op": "between", 
+    "args": [ 
+      { "property": "eo:cloud_cover" }, 
+      [ 0, 50 ] 
+    ]
+  }
+}
+```
+
+### Example 11: Using LIKE 
+
+The LIKE operator allows for pattern-based string matching.
+
+
+    { "op" : "in", "args" : [ { "property" : "prop-name" }, [ 123, 456 ] ] }
+    { "function" : "my-func", "args" : [ "myarg1", 123 ] }
+
+
+#### Example 11: cql2-text (GET)
+
+```http
+filter=mission LIKE "sentinel%"
+```
+
+#### Example 11: cql2-json (POST)
+
+```json
+{
+  "filter": {
+    "op": "like", 
+    "args": [ 
+      { "property": "mission" }, 
+      "sentinel%"
+    ]
+  }
+}
+```
+
+### Example 12: Using Case-insensitive Comparison Functions
+
+The predefined functions `UPPER` and `LOWER` allow for case-insensitive comparisons, and are used in the same way
+user-defined functions are.
+
+#### Example 12: cql2-text (GET)
+
+```http
+filter=LOWER(provider) == "coolsat"
+```
+
+```http
+filter=UPPER(provider) == "NASA"
+```
+
+#### Example 12: cql2-json (POST)
+
+```json
+{
+  "filter": {
+    "op": "eq", 
+    "args": [
+      { 
+        "function" : "lower",
```

Comment:
So upper and lower are implemented as "function" not with "op"?

As these are standard, shouldn't they use "op"? My impression is that "function" was for platform defined functions.

Reply:
Correct. They're not operators since they don't return a boolean value -- they return a string value. However, I looked it up in the spec and I did define them wrong -- they're not operators or functions, they're a separate thing, which is confusing, because the text syntax is exactly the same as if they were functions. I'll update the examples.

The spec is here:
https://github.com/opengeospatial/ogcapi-features/blob/b752479d850c52f372c4164202f041ecbe906e46/cql2/standard/schema/cql2.json

Reply:
I'm going to ask the CQL2 folks about this, because it seems like it would be simpler if it was just a pre-defined function.

Reply:
Filed this issue https://github.com/opengeospatial/ogcapi-features/issues/656

Reply:
ah, so this is changing in CQL2 https://github.com/opengeospatial/ogcapi-features/pull/641

UPPER/LOWER will be replace with a *function* CASEI

Reply:
After looking at CASEI, I have some significant concerns about it. I'm just going to yank it for beta.5, and re-add after it's stabilitzed in CQL2. I wrote up the concerns here if you're interested 
https://github.com/opengeospatial/ogcapi-features/pull/641#issuecomment-985685159